src/Network/Email/Email.php

```diff
@@ -1885,4 +1886,44 @@ protected function _getContentTypeCharset()
         }
         return strtoupper($this->charset);
     }
+
+    /**
+     * Serializes the email object to a value that can be natively serialized and re-used
+     * to clone this email instance.
+     *
+     * @return array Serializable array of configuration properties.
+     */
+    public function jsonSerialize()
+    {
+        $properties = [
+            '_to', '_from', '_replyTo', '_cc', '_bcc', '_subject', '_returnPath', '_readReceipt',
+            '_template', '_layout', '_viewRender', '_viewVars', '_theme', '_helpers', '_emailFormat', '_transport',
```

Comment:
Are you sure it always is a string? (https://github.com/jadb/cakephp/blob/3.0-email-serializer/src/Network/Email/Email.php#L985) I mentioned that concern on a previous version of the PR